dnf.spec

```diff
@@ -118,7 +118,9 @@ Conflicts:      python3-dnf-plugins-extras-common < %{conflicts_dnf_plugins_extr
 %package data
 Summary:        Common data and configuration files for DNF
 Requires:       libreport-filesystem
+%if 0%{?fedora} <= 38
```

Comment:
We want probably opposite there - for Fedora > 38 we require the libdnf5.

Reply:
you are right, my brain has inverted hemispheres. let me check the conditions again